tables/subject_infections/perez-then22-inf.csv

```diff
@@ -0,0 +1,3 @@
+﻿ref_name,subject_name,infection_date_cmp,infection_date,infected_var_name,location,immune_status,severity,section
+Perez-Then22,Yale_mRNA_2dose_d28_CP,~,2020-09-23,Unknown variant,USA,NULL,NULL,NULL
+Perez-Then22,DR_CoronaVac_2dose_mRNA_1dose_d28_CP,~,2020-09-23,Unknown variant,Dominican Republic,NULL,NULL,NULL

```

Comment:
@zachandfox may I ask how you calculate 2020-09-23? I didn't find the date in the paper. If it's 2020-09-23, we may use B.1 as `infected_var_name`

Reply:
In the ethics statement:

> mRNA vaccine BNT162b2 booster were administrated between July 30 and August 27, 2021

So, I set the booster date to August 15th, 2021. Then, in table 1, the "days since last infection" is 326 days, so 326 days before 2021-08-15 is 2020-09-23.